.github/workflows/CD.yaml

```diff
@@ -104,15 +104,18 @@ jobs:
           API_KEY_MANAGER_DB_PASSWORD: ${{ secrets.API_KEY_MANAGER_DB_PASSWORD }}
           INTERNAL_STORAGE_DB_PASSWORD: ${{ secrets.INTERNAL_STORAGE_DB_PASSWORD }}
           DOCUMENT_MANAGER_DB_PASSWORD: ${{ secrets.DOCUMENT_MANAGER_DB_PASSWORD }}
+          QA_SERVICE_DB_PASSWORD: ${{ secrets.QA_SERVICE_DB_PASSWORD }}
           PGADMIN_PASSWORD: ${{ secrets.PGADMIN_PASSWORD }}
           KEYCLOAK_DATALAND_ADMIN_VALUE: ${{ secrets.KEYCLOAK_DATALAND_ADMIN_VALUE }}
           KEYCLOAK_DATALAND_ADMIN_SALT: ${{ secrets.KEYCLOAK_DATALAND_ADMIN_SALT }}
           KEYCLOAK_READER_VALUE: ${{ secrets.KEYCLOAK_READER_VALUE }}
           KEYCLOAK_READER_SALT: ${{ secrets.KEYCLOAK_READER_SALT }}
+          KEYCLOAK_READER_PASSWORD: ${{ secrets.KEYCLOAK_READER_PASSWORD }}
           KEYCLOAK_UPLOADER_VALUE: ${{ secrets.KEYCLOAK_UPLOADER_VALUE }}
           KEYCLOAK_UPLOADER_SALT: ${{ secrets.KEYCLOAK_UPLOADER_SALT }}
```

Comment:
Die Passwörter für die technischen User "admin" und "reviewer" werden hier ja nicht reingegeben.
Das ist aber ok, weil diese technischen User aktuell in keinem Test benutzt werden.

Warum haben wir sie dann in unsere Testumgebung eingebunden?:

![image](https://github.com/d-fine/Dataland/assets/99255296/5275bcb1-f89b-47a2-baf2-f0ab659eb5b2)

Reply:
Warum brauchen wir überhaupt Passwörter für diesen Schritt?
Es wird doch nur das docker compose stack hochgefahren und keycloak initialized. Und für letzteres reichen SALT und VALUE meines Wissens aus.

Reply:
Und im "check deplpoyment" steph am Ende braucht man wahrscheinlich auch nur das "READER" password.
Die anderen sollten nicht notwendig sein, oder?

Reply:
Wird getestet sobald die CI halbwegs grün ist

Reply:
'Die Passwörter für die technischen User "admin" und "reviewer" werden hier ja nicht reingegeben.' Doch.

'Warum brauchen wir überhaupt Passwörter für diesen Schritt?' Agreed WIP.

'Und im "check deplpoyment" steph am Ende braucht man wahrscheinlich auch nur das "READER" password.' Mach ich davon abhängig, wie ausgiebig Cypress die vermissen würde.

Reply:
I consider this done